build.gradle.kts

```diff
@@ -9,14 +9,14 @@ buildscript {
 }
 
 plugins {
-    kotlin("multiplatform") version "1.4.10" apply false
-    id("com.android.application") version "4.1.0" apply false
-    id("com.android.library") version "4.1.0" apply false
+    kotlin("multiplatform") version "1.4.32" apply false
+    id("com.android.application") version "4.1.3" apply false
+    id("com.android.library") version "4.1.3" apply false
```

Comment:
https://youtrack.jetbrains.com/issue/KT-34038 is listed as still open. Do the Android plugins no longer need to come before the multiplatform plugin?

Reply:
Since these are not "applied" here, their order doesn't matter. I believe KT-34038 is still a problem.